File: Java/백준/Bronze/1526. 가장 큰 금민수/가장 큰 금민수.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        while(true){
            int sum=0;
            String b = Integer.toString(a);
            for(int i =0; i<b.length(); i++){
                if(b.charAt(i)=='4' || b.charAt(i)=='7'){
                    sum++;
                }
            }
            if(sum==b.length()){
                System.out.println(b);
                break;
            }
            a--;
        }
    }
}

File: C#/백준/Bronze/1526. 가장 큰 금민수/가장 큰 금민수.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            int sum=0;
            while(true){
                string b = a.ToString();
                for (int i =0; i<b.Length; i++){
                    if(b[i]=='4' || b[i]=='7'){
                        sum++;
                    }
                }
                if(sum==b.Length){
                    Console.WriteLine(b);
                    break;
                }
                sum=0;
                a--;
            }
        }
    }
}